config.toml.example

```diff
@@ -313,11 +313,13 @@
 
 # Whether or not debug assertions are enabled for the compiler and standard
 # library.
-#debug-assertions = debug
+# Defaults to rust.debug value.
```

Comment:
May I suggest consistently adding a blank line before the "Defaults to..." comment (here and elsewhere)? This will help it to standout.

Reply:
Looking at other examples in the file, I doing it the following way would be a good option...

https://github.com/rust-lang/rust/blob/72417d84fb51495a4f1d007fb2397a0b2609ab63/config.toml.example#L127